scripts/ci/notify-test-result.js

```diff
@@ -102,11 +104,25 @@ async function notifyTestResults() {
 
     req.on('error', error => reject(error));
 
+    const data = {
+      testStatus,
+      testUrl: workflowUrl
+    };
+
+    if (versionOrTag) {
```

Comment:
is there a risk that versionOrTag is a truthy non-string? something like `1`

Reply:
I don't think so because the workflow_dispatch input type[ defaults to `string`](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/) if you don't specify a type and I haven't here:
https://github.com/firebase/firebase-js-sdk/blob/d612d6f6e4d3113d45427b7df68459c0a3e31a1f/.github/workflows/e2e-test.yml#L7